src/promptflow/promptflow/_core/operation_context.py

```diff
@@ -30,6 +29,14 @@ def _add_otel_attributes(self, key, value):
         attributes[key] = value
         self[OperationContext._OTEL_ATTRIBUTES] = attributes
 
+    def _remove_otel_attributes(self, keys: list):
+        if isinstance(keys, str):
+            keys = [keys]
+        attributes = self.get(OperationContext._OTEL_ATTRIBUTES, {})
```

Comment:
I think executor should only remove those attributes directly manged by executor?
For attributes like  session, experiment, executor should keep as is?

Reply:
This method only provide the ability to remove attributes, this PR only removes `line_run_id`, what is the concern?

Reply:
Offline synced, there is some misunderstanding about the issue.